apps/glusterfs/block_volume_entry.go

```diff
@@ -376,7 +376,8 @@ func (v *BlockVolumeEntry) Destroy(db wdb.DB, executor executors.Executor) error
 // database operation fails.
 func canHostBlockVolume(tx *bolt.Tx, bv *BlockVolumeEntry, vol *VolumeEntry) (bool, error) {
 	if vol.Info.BlockInfo.FreeSize < bv.Info.Size {
-		logger.Warning("Free size is less than the block volume requested")
+		logger.Warning("Free size %d is less than the requested block volume size %d",
```

Comment:
This hunk updates a log message. So this should go into a separate commit with commit message something like "apps:... impove a log message" ... and update the unit tests that cover this message in the same commit. But nothing more.